README.md

```diff
@@ -196,18 +196,83 @@ end
 
 ### Defining Reactors
 
+Reactors are side affects of events (similar to ActiveRecord callbacks). They are plain old ruby objects and you put the anywhere in your load path with whatever class you prefer. Just define a `#call` class method.
+
+```ruby
+# reactors/user/send_confirmation_email.rb
+
+class User::SendConfirmationEmail
+  def self.call(event)
+    ...
+  end
+end
+```
+
+In your event add a reactors and sync_reactors method that return a class or string, or an array of classes or strings. 
+
+```ruby
+class User::RegisteredEvent
+  ...
+  
+  def reactors
+    "User::SendConfirmationEmail"
+  end
+
+  def async_reactors
+    [User::SendConfirmationEmail, User::SendConfirmationEmail]
+  end
+end
+```
+
```

Comment:
I wrote this readme but never implemented it. Probably not how we'll actually implement.

```suggestion
```